<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNewRecord</class>
 <widget class="QDialog" name="DlgNewRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-BookShop-Desktop_Qt_6_5_0_MinGW_64_bit-Ispravi-greu0161ke/add.png</normaloff>../build-BookShop-Desktop_Qt_6_5_0_MinGW_64_bit-Ispravi-greu0161ke/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-BookShop-Desktop_Qt_6_5_0_MinGW_64_bit-Ispravi-greu0161ke/close.png</normaloff>../build-BookShop-Desktop_Qt_6_5_0_MinGW_64_bit-Ispravi-greu0161ke/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblGenre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGenre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDescription">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtGenre</tabstop>
  <tabstop>txtDescription</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
